src/onegov/town6/templates/tickets.pt

```diff
@@ -84,7 +84,9 @@
                     </tr>
                 </tal:b>
                 <tal:b repeat="ticket tickets">
-                  <div metal:use-macro="layout.macros['tickets_table_row']"/>
+                    <tal:b condition="ticket.state != 'archived'">
```

Comment:
This will mess up the pagination. The collection batch should contain all tickets which are displayed here.